pybotx_widgets/checktable.py

```diff
@@ -0,0 +1,109 @@
+from typing import Any, Dict, List, Optional, TypeVar, Generic, Union
+
+from botx import Bot, Message, MessageMarkup
+from pydantic import root_validator, BaseModel
+
+from pybotx_widgets.resources import strings
+from pybotx_widgets.service import merge_markup, send_or_update_message
+from pybotx_widgets.undefined import Undefined, undefined
+
+T = TypeVar("T")  # noqa: WPS111
+
+
+class CheckboxContent(BaseModel, Generic[T]):
+    """Checkbox content."""
+
+    #: text that will be shown on the element.
+    label: str
+    #: command that will be triggered by click on the element.
+    command: str
+    #: value which will be displayed on the button associated with this checkbox.
+    checkbox_value: Optional[Union[T, Undefined]] = undefined
+    #: list of available values.
+    mapping: Optional[Dict[T, str]] = None
+    #: extra payload that will be stored in button and then received in new message.
+    data: Optional[Dict[str, Any]] = None
+
+    class Config:
+        arbitrary_types_allowed = True
+
+    @root_validator(pre=True)
+    def is_checkbox_value_exist_in_mapping(cls, values):
+        mapping = values.get("mapping")
+        checkbox_value = values.get("checkbox_value")
+        if (
+                mapping
+                and not isinstance(checkbox_value, Undefined)
+                and checkbox_value not in mapping
+        ):
```

Comment:
nit: mb replace with this:
```python
error_conditions = (
    mapping,
    not isinstance(checkbox_value, Undefined),
    checkbox_value not in mapping
)
if all(error_conditions):
    pass

Reply:
In this implementation, if `mapping == None`, an error occurs, since mapping not iterable object.